### PR TITLE
giTag outputs the most recent tag found in refs/tags namespace.

### PR DESCRIPTION
**Pull request by awgn**
Added giTag information for any kind of tag (not only the annotated ones).

### CODE REVIEW

File: src/GitHash.hs [hunk 1]
```diff
@@ -119,6 +121,11 @@ giCommitMessage = _giCommitMessage
 giDescribe :: GitInfo -> String
 giDescribe = _giDescribe
 
+-- | The output of @git describe --always --tags@ for the most recent commit.
+--
```
Comment (snoyberg) [inline src/GitHash.hs:125]: Please add a `@since` comment, ChangeLog entry and version bump

**Review by snoyberg — CHANGES_REQUESTED** [review]

File: src/GitHash.hs (hunk 1 above)
Comment (awgn) [reply src/GitHash.hs:125]: Done

Comment (awgn) [reply src/GitHash.hs:125]: Done

**Review by snoyberg — APPROVED** [review]
Thanks!

**Pull request merged**